backend/nestjs/apps/listing-service/src/rooms/services/rooms.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { CreateRoomDto } from '../dto/create-room.dto';
+import { UpdateRoomDto } from '../dto/update-room.dto';
+import { RoomEntity } from '../entities/room.entity';
+import { RoomRepository } from '../repositories/rooms.repository';
+
+@Injectable()
+export class RoomsService {
```

Comment:
Should we make a base service class to include all the CRUD action for specific entity (in this case RoomEntity) ?